common/src/main/java/org/exoplatform/chat/model/UserBean.java

```diff
@@ -76,11 +76,9 @@ public void setFavorites(List<String> favorites) {
     this.favorites = favorites;
   }
 
-  public boolean isFavorite(String user)
-  {
-    if (favorites!=null)
-    {
-      return (favorites.contains(user));
+  public boolean isFavorite(String room) {
+    if (favorites != null) {
+      return (favorites.contains(room));
     }
     return false;
   }
```

Comment:
there is no really change here

If you want to change something, you can do this : 
```suggestion
  public boolean isFavorite(String user) {
    return favorites!=null && favorites.contains(user);
  }
```

Reply:
Done